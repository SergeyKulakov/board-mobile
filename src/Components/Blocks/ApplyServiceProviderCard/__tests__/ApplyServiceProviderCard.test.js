import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import Component from '../ApplyServiceProviderCard'

describe('ApplyServiceProviderCard', () => {
  const defaultProps = {
    givenName: 'givenName',
    familyName: 'familyName',
    username: 'username',
    avatarURL: null,
    stars: 4,
    idVerified: false,
    isUserJob: false,
    services: [],
    isAcceptLoading: false,
    isAccepted: false,
    onAcceptClick: () => null,
    onRejectClick: () => null,
    onHire: () => null,
    style: undefined,
    onRemoveClick: () => null,
  }
  it('snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
