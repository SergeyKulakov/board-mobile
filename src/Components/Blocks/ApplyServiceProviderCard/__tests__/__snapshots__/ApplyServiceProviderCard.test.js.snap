// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplyServiceProviderCard snapshot 1`] = `
<Styled(View)>
  <Styled(View)>
    <Styled(View)>
      <ProviderPhoto
        avatarURL={null}
        isCheck={false}
        username="username"
      />
    </Styled(View)>
    <Styled(View)>
      <Styled(View)>
        <Styled(Text)>
          givenName F.
        </Styled(Text)>
        <Styled(Icon)
          onClick={[Function]}
        />
      </Styled(View)>
      <Styled(Component)>
        4
      </Styled(Component)>
      <Styled(Text) />
    </Styled(View)>
  </Styled(View)>
</Styled(View)>
`;
